Froger/Frog.java

```diff
@@ -0,0 +1,54 @@
+package Froger;
+
+import javafx.scene.effect.ImageInput;
+import javafx.scene.image.Image;
+import javafx.scene.input.KeyCode;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+
+public class Frog extends Rectangle{
+    private int gridX ;
+    private int gridY ;
+    private GridMap gridMap;
+
+    public Frog(GridMap gridMap){
+
+        this.gridMap = gridMap;
+        this.setWidth(20);
+        this.setHeight(20);
+        this.setFill(Color.GREEN);
+        this.gridX = gridMap.getColumnCount()/2;
+        this.gridY = gridMap.getRowCount()-1;
+        System.out.println(gridX+ " " + gridY);
+        changeLook();
+        addToGrid();
+    }
+    public void jump(KeyCode code){
+        gridMap.getChildren().remove(this);
+
+        if (code == KeyCode.RIGHT){
```

Comment:
jak juz uzzywasz tylu wartosci z enuma to zdecydowanie lepszy bedzie ``switch()``